app/src/main/java/com/example/habitsmasher/ui/dashboard/EditHabitFragment.java

```diff
@@ -61,7 +60,8 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         _reasonText = view.findViewById(R.id.habit_reason_edit_text);
         _dateText = view.findViewById(R.id.habit_date_selection);
 
-        header.setText(HEADER);
+        // Get header from resource file and set it
+        header.setText(getResources().getString(R.string.edit_habit));
```

Comment:
Getting the string from the resource file required the fragment to be in the onCreateView() at least otherwise the context would be null and the app would crash. This prevents this string from being declared in the fields section.